Model/lib/perl/pcbiPubmed.pm

```diff
@@ -30,15 +31,27 @@ sub setPubmedID {
   my $raw_content = LWP::Simple::get ($ncbiEutilsUrl . $pubmed_id);
   return -1 unless $raw_content;    # if HTTP failed
 
-  # Some versions of LWP::Simple (5.827) use $resp->decoded_content , others (1.41) 
+  # Some versions of LWP::Simple (5.827) use $resp->decoded_content , others (1.41)
   # use $resp->content . Encode accordingly.
-  $content = (utf8::is_utf8($raw_content)) ? encode('UTF-8', $raw_content) : $raw_content;  
+  $content = (utf8::is_utf8($raw_content)) ? encode('UTF-8', $raw_content) : $raw_content;
 }
 
 sub fetchPubmedUrl {
     return $publicationPubmedUrl . $id;
 }
-                                                                                             
+
+sub extractAndDecodeTagContent {
+	my ($source, $tag) = @_;
+
+	my $tagContent = EbrcModelCommon::Model::XMLUtils::extractTagContent($source, $tag);
+
+	if ($tagContent =~ /^(?!<).+$/) {
```

Comment:
add a comment here, explaining the regex, and why

Reply:
Updated.